helm/repositories.bzl

```diff
@@ -113,6 +113,45 @@ helm_toolchain_repository = repository_rule(
     },
 )
 
+def _helm_resolved_toolchain_repository(repository_ctx):
```

Comment:
Other repository use a `current_*_toolchain` rule which doesn't require generating a new external repository, is this providing anything more?

E.g. https://github.com/bazelbuild/rules_rust/blob/0.44.0/rust/private/toolchain_utils.bzl#L80-L97

Reply:
Good question. I was following an existing pattern I had seen, but this way you've linked seems simpler. I'll give it a try.

Reply:
Updated to use the pattern you suggested.